app/forms/BusinessRegistrationForms.scala

```diff
@@ -197,10 +198,13 @@ object BusinessRegistrationForms {
       _.isEmpty
     }
     
-    val validatePostCode = ApplicationConfig.validateNonUkClientPostCode(service) && !isAgent
-    if (postCode.isEmpty && validatePostCode) {
+    val postCodeFlag = ApplicationConfig.validateNonUkClientPostCode(service) && !isAgent
```

Comment:
Please name it appropriately.

Reply:
reverted back to how it was (validatePostCode)